controllers/pipelines/suite_decoupled_test.go

```diff
@@ -101,6 +101,12 @@ var _ = BeforeSuite(func() {
 	Expect((&pipelinesv1.RunConfiguration{}).SetupWebhookWithManager(k8sManager)).To(Succeed())
 	Expect((&pipelinesv1.Run{}).SetupWebhookWithManager(k8sManager)).To(Succeed())
 
+	provider := pipelinesv1.RandomProvider()
+	provider.Name = testConfig.DefaultProvider
+	provider.Namespace = testConfig.WorkflowNamespace
+	err = k8sClient.Create(ctx, provider)
+	Expect(err).NotTo(HaveOccurred())
```

Comment:
```suggestion
	Expect(k8sClient.Create(ctx, provider)).To(Succeed())
```

Reply:
changed in [7c2bb14](https://github.com/sky-uk/kfp-operator/pull/354/commits/7c2bb142f5f62be56f3935bfffb7f65deffb5fb7)